pages/outreachy.md

```diff
@@ -189,7 +189,7 @@ Monitor communication channels and issues and add new topics to the [FAQ](https:
 
 OpenTracing and Jaeger participated in the following Outreachy programs:
 
-* [December 2017 to March 2018](/outreachy-201702)
+* [December 2017 to March 2018](https://github.com/opentracing/opentracing.io/blob/master/pages/outreachy-201702.md)
```

Comment:
we don't want to link to github files, this file should be rendered as part of the website. Not sure why it doesn't - maybe the generator doesn't like the `-201702` suffix

Reply:
@yurishkuro Thanks for reviewing. :-)
Should I revert back this change?

Reply:
better to find a solution why this page doesn't show under /outreachy-201702